experimental/traffic-portal/src/app/core/profiles/profile-table/profile-table.component.ts

```diff
@@ -115,11 +115,11 @@ export class ProfileTableComponent implements OnInit {
 				}
 			}
 		);
-		const ID = this.route.snapshot.paramMap.get("id");
-		if (ID === null) {
+		const hasParameter = this.route.snapshot.queryParamMap.get("hasParameter");
+		if (hasParameter == null) {
```

Comment:
This is an interesting corner case. Apparently, the Angular default linter settings don't use the default `eqeqeq` rule, and allow you to use either style of comparison with `null` and only with `null`. _Technically_, there is a case where this would fail to produce the same result as `===`, but it's not possible given the type of `hasParameter` (specifically `null == undefined` is `true` but `null === undefined` is `false`). So, since the linter doesn't care, neither do I, but I'm making a mental note to correct that exclusion.